src/util.cc

```diff
@@ -1,5 +1,6 @@
 #include "util.h"
 
+#include "util/compression.h"
```

Comment:
how about adding rocksdb code path with a leading `rocksdb`, so we can distinguish them easily with Titan's.

Reply:
It doesn't work like that. "ROCKSDB_DIR" and "ROCKSDB_DIR/include" are added as project include dirs. Files like "util/compression.h" reside under ROCKSDB_DIR. To include them with a leading "rocksdb", you have to include "ROCKSDB_DIR/..", which is not what we wanted.

And it's actually easy to distinguish (so far): `#include "xx.h"` and `#include "titan/xx.h"` is titan's, others are all rocksdb's.